src/main/java/mate/academy/spring/dto/request/MovieSessionRequestDto.java

```diff
@@ -1,10 +1,14 @@
 package mate.academy.spring.dto.request;
 
 import java.time.LocalDateTime;
+import javax.validation.constraints.NotNull;
 
 public class MovieSessionRequestDto {
+    @NotNull
     private Long movieId;
+    @NotNull
     private Long cinemaHallId;
+    @NotNull
     private LocalDateTime showTime;
```

Comment:
```suggestion
    @NotNull
    @JsonFormat(pattern = DateTimePatternUtil.DATE_TIME_PATTERN)
    private LocalDateTime showTime;
```
without that pattern, your DTO could receive a date only in this non-client-friendly format: "2020-12-21T21:00:00.000Z"